index.html

```diff
@@ -1,317 +1,272 @@
-                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
-                                has been the industry's standard
-                            </p>
-                            <ul class="description-tags">
-                                <li>html</li>
-                                <li>Bootstrap</li>
-                                <li>Ruby</li>
-                            </ul>
-                        </div>
-                        <button class="btn btn-card" type="button" >See Project</button>   
-                    </li>              
+        <ul class="headline-social-media">
+          <li>
+            <a href="#">
+              <img src="./icons/Disabledgithub-icon.png" alt="gitHub icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                src="./icons/DisabledLinkedIn-icon.png"
+                alt="LinkedIn Icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                src="./icons/Enabledtwo-fingers-icon.png"
+                alt="Two fingers Icons"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./icons/Disabledtwitter-icon.png" alt="Twitter Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./icons/DisabledM-icon.png" alt="Make Icon" />
+            </a>
+          </li>
+        </ul>
+      </section>
 
-                </ul>               
+      <section id="recent-works">
+        <div class="recent-works-header">
+          <h2>My Recent Works</h2>
+          <div class="thin-line"></div>
+        </div>
+
+        <div class="my-works">
+          <div class="featured">
+            <div class="first-card-image-container">
+              <img
+                class="first-card-img-mobile"
+                src="./images/mobile-first-card.svg"
+                alt="project image"
+              />
+              <img
+                class="first-card-img-desktop"
+                src="./images/Desktop-firstcard-img.svg"
+                alt="project image"
+              />
+            </div>
+            <div class="first-card">
+              <h3>Multi-Post Stories</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <ul>
+                <li class="tags">css</li>
+                <li class="tags">html</li>
+                <li class="tags">Bootstrap</li>
+                <li class="tags">Ruby</li>
+              </ul>
+              <button type="button" class="btn btn-feature">See Project</button>
```

Comment:
- [ ] Currently, this button is not working as intended. Kindly make sure that it causes a modal with data for that specific project to open